doc/usecases/mount/UC_delete_from_file.md

```diff
@@ -0,0 +1,52 @@
+# Use Case: Delete configuration data from a file
```

Comment:
I would simplify this and only have a get and set use case. (Delete is part of set).
I think it is enough that you only write the rdb use cases.
I don't think they should be part of `mount`, but rather another folder ("rdb").